packages/storybook8/stories/Concepts/MediaAccess/snippets/MediaAccessRemoteParticipants.snippet.tsx

```diff
@@ -0,0 +1,15 @@
+import { VideoGallery, usePropsFor } from '@azure/communication-react';
+import React from 'react';
+
+export const CallScreen = (): JSX.Element => {
+  // Use usePropsFor to get properties for VideoGallery
+  const videoGalleryProps = usePropsFor(VideoGallery);
+
+  // Logging remote participants' Media access state without modifying the array
+  videoGalleryProps.remoteParticipants.forEach((participant) => {
+    console.log('Participant media access:', participant.mediaAccess);
```

Comment:
nit: This is a nice snippet but I would be more specific with the logging of media access with something like.
```
    console.log(
      `Participant [${participant.userId}:${participant.displayName}]'s media access:`,
      participant.mediaAccess
    );
```

Reply:
updated logging